src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/panels/SolutionListPane.java

```diff
@@ -46,51 +46,47 @@ public void init(){
 
         refreshSolutionList();
 
-        myJpanel.add(
+        mainPanel.add(
                 scrollPane,
                 new CustomConstraints(0, 0, GridBagConstraints.WEST, 1.0, 0.8, GridBagConstraints.BOTH)
         );
 
-        JButton newSolutionButton = new JButton("Add solution");
-        newSolutionButton.addActionListener(
-                new ActionListener() {
+        JButton newSolutionButton = new JButton("Add Solution");
```

Comment:
same here as well

Reply:
there is no difference i just kept new ActionListener() in the same line as newSolutionButton.addActionListener. there is no difference in functionality